src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,92 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static final String VALID_LOGIN = "testUser";
+    private static final String VALID_PASSWORD = "123user";
+    private static final String INVALID_PASSWORD = "123";
+    private static final Integer VALID_AGE = 20;
+    private static final Integer INVALID_AGE = 15;
+
+    private static RegistrationService registrationService;
+    private static User user;
+
+    @BeforeAll
+    public static void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    public void userInit() {
+        user = new User(VALID_LOGIN, VALID_PASSWORD, VALID_AGE);
+    }
+
+    @Test
+    public void register_validUser_Ok() {
+        User actual = registrationService.register(user);
+        assertEquals(user, actual);
+        StorageDao storageDao = new StorageDaoImpl();
```

Comment:
we could also introduce a field instead of local variable, because we use `StorageDao` in few places